getting_started/end_to_end_tutorials.rst

```diff
@@ -0,0 +1,20 @@
+.. _application examples:
+
+Application examples
+--------------------
+
+These are standalone tutorials that can be followed in any order.
+
+**Coming soon:**
+
+* Liquid state machine: Another classical canonical example in neuromorphic computing, illustrating a complex, spiking and recurrent neural network for time series classification.
+
+* Learning: Many neuromorphic architectures offer local on-chip learning via bio-inspired STDP learning. This tutorial provides a simple example how such learning rules can be used.
```

Comment:
I suggest taking these out until we have the content ready. This gives me "Site under construction" vibes.

Reply:
I grouped all tutorials together and removed the "coming soon" parts.